examples/k8s_cron_job.yaml

```diff
@@ -35,6 +35,8 @@ spec:
               value: opencost.opencost.svc.cluster.local.
             - name: OPENCOST_PARQUET_SVC_PORT
               value: "9003"
+            - name: OPENCOST_PARQUET_STORAGE_BACKEND
```

Comment:
Isn't this supposed to be a string with a single backend ? 
you can add as a comment that it can be either aws or azure